applications/asset_tracker_v2/src/cloud/cloud_codec/json_common.c

```diff
@@ -1008,6 +1008,18 @@ int json_common_config_add(cJSON *parent, struct cloud_data_cfg *data, const cha
 		json_add_obj_array(nod_list, ncell_str);
 	}
 
+	if (data->no_data.wifi) {
+		cJSON *wifi_str = cJSON_CreateString(CONFIG_NO_DATA_LIST_WIFI);
```

Comment:
CONFIG_NO_DATA_LIST_WIFI probably needs to be added to the json_protocol_names for azure as well.